lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -11,7 +11,6 @@
 
 from capa.tests.response_xml_factory import StringResponseXMLFactory
 from courseware.courses import get_course_by_id  # pyline: disable=import-error
-from courseware.field_overrides import OverrideFieldData  # pylint: disable=import-error
 from courseware.tests.factories import StudentModuleFactory  # pylint: disable=import-error
 from courseware.tests.helpers import LoginEnrollmentTestCase  # pylint: disable=import-error
```

Comment:
I am not sure: I did not touch that code, but I'll make sure they are needed and, if not, I'll remove them